mcstatus/__main__.py

```diff
@@ -22,10 +25,33 @@ def _motd(motd: Motd) -> str:
 
 
 def ping(server: SupportedServers) -> int:
-    # this method supports both Java and Bedrock.
-    # only Java supports the `ping` packet, and even then not always:
-    # https://github.com/py-mine/mcstatus/issues/850
-    print(f"{server.status().latency}")
+    notsup = "notsup"
```

Comment:
Wouldn't this be better?

```py
def ping(server: SupportedServers) -> int:
    if isinstance(server, BedrockServer):
        # bedrock doesn't have ping method
        print(f"{server.status().latency}")
        return 0

    try:
        latency = server.ping()
    except Exception as exception:
        address = f"{server.address.host}:{server.address.port}"
        warnings.warn(
            f"contacting {address} failed with a 'ping' packet but succeeded with a 'status' packet,\n  "
            f"this is likely a bug in the server-side implementation.\n  "
            f"for more details, see: https://mcstatus.readthedocs.io/en/stable/pages/faq/\n",
            stacklevel=1,
        )

        latency = server.status().latency

    print(f"{latency}")
    return 0
```

Yes it has some code duplication, but it is much more readable